### ISSUE TITLE
gr_modtool rename: Renaming of pattern instead of entire blockname

### ISSUE THREAD

**Issue by swap-nil7** [gnuradio/gnuradio#1727]
While using ```gr_modtool rename``` the script searches for instances of the same pattern in all the file names in different directories instead of searching for the exact block name. For example:-
```
Enter name of block/code to rename (without module name prefix): mul
Block/code to rename identifier: mul
Enter name of block/code (without module name prefix): sqr
```
will also rename the files corresponding to block name ```multiplexer``` which is undesirable!

**Comment by dkozel** [comment]
@swap-nil7 Will you be addressing this issue as part of your work?

**Comment by swap-nil7** [comment]
@dkozel: Yes, I will do it. :+1:

**Comment by marcusmueller** [comment]
fixed in #1873
